pages/model-sets/[id].js

```diff
@@ -243,9 +244,6 @@ export async function getServerSideProps({ params, req, query }) {
     await request.getObject(`/model-sets/${params.id}/`)
   ).union();
   if (FetchRequest.isResponseSuccess(modelSet)) {
-    const softwareVersion = modelSet.software_version
-      ? (await request.getObject(modelSet.software_version)).optional()
-      : null;
     const documents = modelSet.documents
```

Comment:
Delete the `softwareVersion` in the returned props of this function. It’ll probably crash when you visit a ModelSet page.

Reply:
ok